### DISCUSSION TITLE
Supabase Missing Cache

### DISCUSSION THREAD

**Discussion by pedro123456789-wq** [elizaOS/eliza#684] [Q&A]
Hello everyone, 
When I try to use supabase I get an error because it does not implement the IDatabaseCacheAdapter class. 
Is there a way around this?
Thanks

**Comment by ealeksandrov7** [comment]
I guess not.. The supabase adapter has a bunch of issues currently. I think you have to implement your own caching mechanism for now

**Comment by SnoochiMink** [comment]
> Hello everyone, 
> 
> When I try to use supabase I get an error because it does not implement the IDatabaseCacheAdapter class. 
> 
> Is there a way around this?
> 
> Thanks

Did you get a fix for this? If so how?
